irohad/main/irohad.cpp

```diff
@@ -83,6 +83,8 @@ DEFINE_validator(keypair_name, &validate_keypair_name);
  */
 DEFINE_bool(overwrite_ledger, false, "Overwrite ledger data if existing");
 
+DEFINE_bool(reuse_state, false, "Try to use existing data in WSV at startup.");
```

Comment:
Consider replacing the term "WSV" which may not be very clear to most users with a ubiquitous and self-explanatory "state"